ethereum_history_api/circuits/lib/src/fixtures/mainnet/london/crypto_punks/storage_proof.nr

```diff
@@ -0,0 +1,4 @@
+use crate::account_with_storage::StorageProof;
+
+global proofs = [
+];
```

Comment:
empty?

Reply:
Good catch. Removed. Was caused by a leftover empty list in fixtures config